src/doc/rustdoc/src/unstable-features.md

```diff
@@ -467,3 +467,20 @@ $ rustdoc src/lib.rs -Z unstable-options --runtool valgrind
 ```
 
 Another use case would be to run a test inside an emulator, or through a Virtual Machine.
+
+### `--source-code-external-url`: using external source code
+
+In case you don't to generate the source code files and instead use existing ones (available
+through an HTTP URL!), you can use this option:
+
+```bash
+$ rustdoc src/lib.rs -Z unstable-options --source-code-external-url 'https://somewhere.com'
```

Comment:
Maybe it's a good idea to use an example domain for the purpose of being used in examples, like https://example.com as per https://www.iana.org/domains/reserved.

Reply:
I don't see the difference with what I wrote?

Reply:
`somewhere.com` could be owned by anyone, and they could put any content there. `example.com` on the other hand is registered by the Internet Assigned Numbers Authority, and it's safe to use in examples.

Reply:
That doesn't make sense to use it directly like this... Originally, I wanted to put `http://a.a`...

Reply:
I'm not sure where the misunderstanding is, but put simply, example.com is safe to use in examples, while any other random domain name is not (because anyone can grab those and host any content; example.com will always be safe). Sorry about the noise.

Reply:
I still don't really understand your point... Like I said, if you use it as is, it'll just not do anything except creating links to "somewhere.com". Which isn't what any user want. If you're still not convinced, I propose you the following: open a PR changing the domain I'm pointing to in the example. I'll merge it without problem. What do you think?

Reply:
Okay, that sounds good to me if we can settle there.

Reply:
Perfect! Then once this is merged, please open a PR and ping me on it so it can get merged quickly.